src/WebJobs.Script.WebHost/Scale/TableEntityConverter.cs

```diff
@@ -96,6 +96,7 @@ public static bool TryGetEntityProperty(JProperty property, out EntityProperty e
                     entityProperty = new EntityProperty(value.ToObject<long>());
                     return true;
                 case JTokenType.String:
+                case JTokenType.TimeSpan:
```

Comment:
We recently moved to the latest version of Azure Storage SDK, and ideally we'd be taking advantage of [TableEntity.Flatten](https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.cosmos.table.tableentity.flatten?view=azure-dotnet) which was newly added, rather than this custom conversion code we had to write.

To limit the scope of the changes for this fix, I won't make those changes here, since we have to be careful. I've logged https://github.com/Azure/azure-functions-host/issues/6914 for that.